bundler/spec/bundler/bundler_spec.rb

```diff
@@ -191,6 +187,7 @@
 
     shared_examples_for "it returns the correct executable" do
       it "returns the expected file" do
+        skip "respective bat file need to exist on Windows" if Gem.win_platform?
```

Comment:
I also see a lot of new skips in `bundler/spec/commands/exec_spec.rb`. We should keep what's working now, working in the same way.

Reply:
I will investivate this when I have time.

Regarding new skips, It has no meaning when use `echo` or `cat` because they are not executable on Windows.

![image](https://github.com/user-attachments/assets/96a0775f-b320-498d-89f4-c7870db4d5b1)

I don't know how these tests works originally but debugging in the cloud is outside my ability.

Other than that I think there are no new new skips beacuase some are nested, which are skipped in a group.

Reply:
well for this I can say the logic is now changed. For `which` to return the executable either we need exe or bat suffix or a respective bat file need to exist. So this test no long works. 

But I also have trouble understanding the mocks so I cannot fix them.